app/javascript/crayons/MultiInput/__stories__/MultiInput.mdx

```diff
@@ -0,0 +1 @@
+# Multi input
```

Comment:
Quick, non-blocking suggestion: let's capitalize the "i" here and hyphenate the word! I'm okay with holding off on changing this until when the rest of the documentation is tackled, so that this tiny comment doesn't hold up the PR. 😊 
```suggestion
# Multi-Input
```